tests/unit/test_reset_instance_action.py

```diff
@@ -3,7 +3,7 @@
 
 """Reset instance action unit tests."""
 
-# pylint: disable=protected-access
+# pylint: disable=protected-access, no-member
```

Comment:
Which is the problematic line here?

Reply:
They are these 2 lines : 
- https://github.com/canonical/synapse-operator/blob/c5fa04fa044ca89e11b0513861ae7d6d1432eb05/tests/unit/test_reset_instance_action.py#L54
- https://github.com/canonical/synapse-operator/blob/c5fa04fa044ca89e11b0513861ae7d6d1432eb05/tests/unit/test_reset_instance_action.py#L155

We have used `call_args` in tests in the past but I don't really know why this lint error is only now showing

Reply:
Perhaps the return value of  `call_args`?

Reply:
I have moved the pylint comments to the right place